tests/src/JIT/Methodical/Arrays/misc/arrres.cs

```diff
@@ -62,52 +62,52 @@ private static void Test2()
                 s_arr[i].CheckValid();
                 s_arr[i] = null;
             }
-            GC.Collect();
-            GC.WaitForPendingFinalizers();
-            GC.Collect();
         }
         [System.Runtime.CompilerServices.MethodImplAttribute(System.Runtime.CompilerServices.MethodImplOptions.NoInlining)]
         private static void Test3()
         {
+            GC.Collect();
+            GC.WaitForPendingFinalizers();
+            GC.Collect();
             for (int i = 0; i < 50; i++)
```

Comment:
Would it make more sense to create a new `DoCollect()` function that does these three lines, is marked NoInlining, and is called between the Test1()/Test2()/etc. calls in Main()?